FirebaseCombineSwift/Tests/Unit/Functions/HTTPSCallableTests.swift

```diff
@@ -0,0 +1,140 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+import Combine
+import XCTest
+@testable import FirebaseFunctions
+
+class MockFunctions: Functions {
+  var mockCallFunction: () throws -> HTTPSCallableResult?
+  override func callFunction(_ name: String,
+                             with data: Any?,
+                             timeout: TimeInterval,
+                             completion: @escaping (HTTPSCallableResult?, Error?) -> Void) {
+    do {
+      let result = try mockCallFunction()
+      completion(result, nil)
+    } catch {
+      completion(nil, error)
+    }
+  }
+}
+
+class HTTPSCallableTests: XCTestCase {
+  func testCallWithoutParametersSuccess() {
+    // given
+    var cancellables = Set<AnyCancellable>()
+    let funcationWasCalledExpectation = expectation(description: "Function was called")
+
+    let functions = MockFunctions.functions()
+    let expectedResult = "mockResult w/o parameters"
+    functions.mockCallFunction = {
+      HTTPSCallableResult(data: expectedResult)
```

Comment:
nit: I would suggest adding a specific expectation to validate that the method was called and called only once. It may seem obvious and redundant now as the implementation is pretty simple, but may protect us from unintended typos or accident changes in the future.

Reply:
I'm going to use assertForOverFulfill for this.